### PR TITLE
add extra info to authentication

### PR DESCRIPTION
**Pull request by samundra**
- add extra information to cookie-auth example.

The render happens on server and client. So, based on where the page is being rendered, the logic for retrieving token varies. The comment will be helpful for people who are just starting with next.js authentication flow.

### CODE REVIEW

File: examples/with-cookie-auth/www/utils/auth.js [hunk 1]
```diff
@@ -63,12 +63,20 @@ export const withAuthSync = WrappedComponent =>
...
+   * means user is not logged in.
+   */
   if (ctx.req && !token) {
     ctx.res.writeHead(302, { Location: '/login' })
     ctx.res.end()
     return
   }
 
+  /*
+   * We already checked for server. This should only happen on client.
```
Comment (timneutkens) [inline examples/with-cookie-auth/www/utils/auth.js]:
```suggestion
// We already checked for server. This should only happen on client.
```

File: examples/with-cookie-auth/www/utils/auth.js [hunk 2]
```diff
@@ -63,12 +63,20 @@ export const withAuthSync = WrappedComponent =>
...
+   * rendered on server. If we are on server and token is not available,
+   * means user is not logged in.
+   */
   if (ctx.req && !token) {
     ctx.res.writeHead(302, { Location: '/login' })
     ctx.res.end()
     return
   }
 
+  /*
```
Comment (timneutkens) [inline examples/with-cookie-auth/www/utils/auth.js]:
```suggestion

```

File: examples/with-cookie-auth/www/utils/auth.js [hunk 3]
```diff
@@ -63,12 +63,20 @@ export const withAuthSync = WrappedComponent =>
...
+   * We already checked for server. This should only happen on client.
+   */
```
Comment (timneutkens) [inline examples/with-cookie-auth/www/utils/auth.js]:
```suggestion

```

**Comment by winhackger** [comment]
excuse me, this project is already finish? Can I follow this project directly?

**Comment by Timer** [comment]
Hi @winhackger! Thanks for reaching out.

If you'd like to contribute, please look at issues tagged [`good first issue`](https://github.com/zeit/next.js/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22). We look forward to your contributions.

If this isn't what you meant, could you please elaborate more? 😄

**Comment by winhackger** [comment]
Am want to learn from this project.
Now am start project with nextjs 😅

**Comment by Timer** [comment]
@winhackger using https://nextjs.org/learn would be the best way to learn Next.js for your project!

**Comment by winhackger** [comment]
@Timer 
am try to login have some error on console.

Error! { Error: connect ECONNREFUSED 127.0.0.1:3001
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1104:14)
  errno: 'ECONNREFUSED',
  code: 'ECONNREFUSED',
  syscall: 'connect',
  address: '127.0.0.1',
  port: 3001 }


how to past ?

**Comment by samundra** [comment]
@winhackger Looks like you have connection issues. `ECONNREFUSED` basically means your server is refusing connection on port 3001. This is not related to the current auth example. You have to make your server auth endpoint reachable. 

I would suggest the following:

1) Make sure that your server is running fine with correct endpoints.
2) Try to reach endpoint and see if it returns correct auth response or not.

**Comment by winhackger** [comment]
Thank you. I’m forget

**Pull request merged**